examples/window-app/silabs/efr32/args.gni

```diff
@@ -24,4 +24,4 @@ chip_enable_openthread = true
 openthread_external_platform =
     "${chip_root}/third_party/openthread/platforms/efr32:libopenthread-efr32"
 
-use_base_app = false
+use_base_app = true
```

Comment:
This build option should be removed at large since it's not needed anymore

Reply:
Note: so far, I need to specify use_base_app = true when building the application through the terminal:

`./scripts/examples/gn_efr32_example.sh ./examples/window-app/silabs/efr32/ ./out/window-app BRD4187C use_base_app=true`